charts/kueue/values.yaml

```diff
@@ -29,8 +29,13 @@ controllerManager:
         memory: 512Mi
     podSecurityContext:
       runAsNonRoot: true
+      seccompProfile:
```

Comment:
Please also update the [manager.yaml](https://github.com/kubernetes-sigs/kueue/blob/4c2a0cddc35b80995653c24a9fe85cb57743179f/config/components/manager/manager.yaml#L18).

Reply:
Thanks